README.md

```diff
@@ -1 +1,77 @@
-# tictactoe-tdd-java
+# tictactoe-tdd-java (WIP)
+
+Implementing Tic-Tac-Toe using a Java
+
+## Getting Started
+
+prerequisite: [Java Runtime Environment (JRE)](https://www.java.com/en/download/manual.jsp) must be installed onto your computer.
+
+- Install Java Open SDK 17
+- Brew Install Gradle
+- Disable Kotlin in IDE
```

Comment:
Kinda a nitpick, but It might help to pointto links of where to download, commands to run, or which menu items to look for these things